_authors/tkharban.markdown

```diff
@@ -0,0 +1,8 @@
+---
+name: Tushar Kharbanda
+short_name: tkharban
+photo: '/assets/media/authors/tkharban.jpg'
+linkedin: 'tushar-kharbanda'
+---
+
+**Tushar** is Software Engineer at AWS OpenSearch. In his spare time he is also an active contributor to OpenSearch.
```

Comment:
- "AWS OpenSearch" -> "OpenSearch at Amazon Web Services" 
- "In his spare time he is also an active contributor to OpenSearch." ... maybe something that rounds you out more :)